governance/votes/Executive vote - December 10, 2021.md

```diff
@@ -0,0 +1,97 @@
+
+### Replacing MKR Vesting Streams
+
+As per this successful [governance poll](https://vote.makerdao.com/polling/QmYdDTsn?network=mainnet#poll-detail), the following MKR Vesting Streams will be canceled and replaced by streams funded by the Protocol Treasury (Pause Proxy) if this executive proposal passes.
+
+1,051.25 MKR will be streamed to the **[Oracles (ORA-001) Core Unit](https://mips.makerdao.com/mips/details/MIP39c2SP13)** starting July 1, 2021 through July 1, 2022 at **0x2d09B7b95f3F312ba6dDfB77bA6971786c5b50Cf** in accordance with their active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP15). A 1-year cliff is in place, making MKR claimable on July 1, 2022.
+
+803.18 MKR will be streamed to the **[Growth (GRO-001) Core Unit](https://mips.makerdao.com/mips/details/MIP39c2SP4)** starting July 1, 2021 through July 1, 2022 at **0x7800C137A645c07132886539217ce192b9F0528e** in accordance with their active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP19). A 1-year cliff is in place, making MKR claimable on July 1, 2022.
+
+13,930 MKR total will be streamed to 14 team members of the **[Protocol Engineering (PE-001) Core Unit](https://mips.makerdao.com/mips/details/MIP39c2SP7)** as detailed below, in accordance with their active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP7). All team members have a 1-year cliff and will have their 995 MKR streamed over 4 years.
+
+| **Address**                                | **Stream Start Date** |
+|--------------------------------------------|-----------------------|
+| 0xfDB9F5e045D7326C1da87d0e199a05CDE5378EdD | 2021-05-01            |
+| 0xBe4De3E151D52668c2C0610C985b4297833239C8 | 2021-05-01            |
+| 0x58EA3C96a8b81abC01EB78B98deCe2AD1e5fd7fc | 2021-05-01            |
+| 0xBAB4Cd1cB31Cd28f842335973712a6015eB0EcD5 | 2021-05-01            |
+| 0xB5c86aff90944CFB3184902482799bD5fA3B18dD | 2021-05-01            |
+| 0x780f478856ebE01e46d9A432e8776bAAB5A81b5b | 2021-05-01            |
+| 0x34364E234b3DD02FF5c8A2ad9ba86bbD3D3D3284 | 2021-05-01            |
+| 0x46E5DBad3966453Af57e90Ec2f3548a0e98ec979 | 2021-05-01            |
+| 0x18CaE82909C31b60Fe0A9656D76406345C9cb9FB | 2021-05-01            |
+| 0x301dD8eB831ddb93F128C33b9d9DC333210d9B25 | 2021-05-01            |
+| 0xBFC47D0D7452a25b7d3AA4d7379c69A891bD5d43 | 2021-05-01            |
+| 0xcD16aa978A89Aa26b3121Fc8dd32228d7D0fcF4a | 2021-09-13            |
+| 0x3189cfe40CF011AAb13aDD8aE7284deD4CD30602 | 2021-06-21            |
+| 0x29b37159C09a65af6a7CFb062998B169879442B6 | 2021-09-20            |
+
+For more information on the rationale for this streaming change, please see this [thread](https://forum.makerdao.com/t/signal-request-should-we-mint-new-mkr-for-protocol-expenses-or-fund-them-with-mkr-from-the-makerdao-treasury/11629) to inform your position before voting.
```

Comment:
Another perfectly dismissable trifle: [this thread] sounds/reads a bit better to me.

Reply:
I'm with you, I think Long likes shorter hyper links... but since he's off ;)